FEWD-Assignment3/FEWD-Assignment3.css

```diff
@@ -0,0 +1,290 @@
+*
+Project Name: Relaxr Landing Page
+Client: Relaxr
+Author: Thalia Wiggins
+Developer @GA in DC
+*/
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+    -moz-box-sizing: border-box; /* Firexfox */
+    -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+    box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+    content: ".";
+    display: block;
+    clear: both;
+    visibility: hidden;
+    line-height: 0;
+    height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+.alignright {
+ float: right;
+ padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+ float: left;
+ padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
```

Comment:
To answer your question about renaming FEWD file, I am still using and editing in VSC  (and paste it here as you see,) and those are some of the suggestions I found when i was looking on how to do the assignment. I don't know if its form W3 or stackflow, etc., since I looked at them all. I do not mean to over do it, but I still am 50/50 confident in my abilities, and since it is I who gets confused at the end :( I'm sorry and will continue to edit in github.